modules/service/src/main/scala/ch/epfl/bluebrain/nexus/admin/service/encoders/organization.scala

```diff
@@ -0,0 +1,40 @@
+package ch.epfl.bluebrain.nexus.admin.service.encoders
+
+import cats.syntax.show._
+import ch.epfl.bluebrain.nexus.admin.core.resources.Resource
+import ch.epfl.bluebrain.nexus.admin.ld.Const.{nxv, projectContext, schema}
+import ch.epfl.bluebrain.nexus.admin.ld.JsonLD._
+import ch.epfl.bluebrain.nexus.admin.refined.ld.Namespace
+import ch.epfl.bluebrain.nexus.admin.refined.organization.OrganizationReference
+import io.circe.{Encoder, Json}
+
+object organization {
+
+  /**
+    * Constructs implicit encoder for organization resource which creates JSON-LD representation from underlying RDF graph.
+    * @param organizationNamespace organization namespace used to create `@id`
+    * @return encoder for organization resource
+    */
+  implicit def organizationEncoder(
+      implicit organizationNamespace: Namespace): Encoder[Resource[OrganizationReference]] =
+    Encoder.encodeJson.contramap { resource =>
+      val ld =
+        resource.value
+          .appendContext(projectContext)
+
+      Json.obj(
+        "@id"   -> Json.fromString(s"${organizationNamespace.value}${resource.id.value.value}"),
```

Comment:
Better to use the properties form the `Const.scala` (`nxv.rev.reference.value`, `@type`, `@id`), then we have to change them only in one place if we have to.
Also, it is used like that in other parts of the code...so it doesn't seem consistent.